influxdb_client/client/query_api.py

```diff
@@ -146,10 +156,53 @@ def query_data_frame_stream(self, query: str, org=None, data_frame_index: List[s
 
     # private helper for c
     @staticmethod
-    def _create_query(query, dialect=default_dialect):
-        created = Query(query=query, dialect=dialect)
+    def _create_query(query, dialect=default_dialect, params: dict = None):
+        created = Query(query=query, dialect=dialect, extern=QueryApi._build_flux_ast(params))
         return created
 
+    @staticmethod
+    def _params_to_extern_ast(params: dict) -> List['OptionStatement']:
+
+        statements = []
+        for key, value in params.items():
+
```

Comment:
Parameters without defined value should be skipped.

```suggestion
        if value is None:
            continue
```